crates/noirc_frontend/src/hir_def/types.rs

```diff
@@ -254,7 +259,7 @@ impl std::fmt::Display for Type {
                 let elements = vecmap(elements, ToString::to_string);
                 write!(f, "({})", elements.join(", "))
             }
-            Type::Bool => write!(f, "bool"),
+            Type::Bool(is_const) => write!(f, "{}bool", is_const),
```

Comment:
I think there needs to be a space: "{} bool"

Reply:
IsConst will print "const " if const or "" otherwise so there is no space needed here